src/node_dotenv.cc

```diff
@@ -134,13 +134,25 @@ void Dotenv::ParseContent(const std::string_view input) {
       auto newline = content.find('\n');
       if (newline != std::string_view::npos) {
         content.remove_prefix(newline + 1);
+        content = trim_spaces(content);
         continue;
+      } else {
+        break;
```

Comment:
Please add a comment here. Explain it as much as you can. Possibly with examples so it is more readable/understandable to reviewers/contributors.